Assets/Scripts/Model/Timing/AccessCard.cs

```diff
@@ -51,20 +51,7 @@ async private Task Trash()
 
         async private Task Steal()
         {
-            if (card.Stealable != Stealable.CANNOT_STEAL)
-            {
-                card.FlipFaceUp();
-                game.runner.zones.score.Add(card, 2);
-                UnityEngine.Debug.Log("Stole CST!");
-
-                /**
-                    Maybe each card can respond to being stolen:
-                    - you must steal me (Project Atlas)
-                    - you can steal me if you pay me (Obokata Protocol)
```

Comment:
Net damage is a new story, but paying credits is already covered.
So I want to add Bellona to support cost-to-steal case.

- [ ] Bellona

Reply:
I don't know that card, I have to have a look at it